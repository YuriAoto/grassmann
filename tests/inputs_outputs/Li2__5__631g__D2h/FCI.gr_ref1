Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__631g__D2h

Command:
../../../Grassmann --method CCD_mani_minD --molpro_output FCI.out

Starting at 26 May 2021 - 20:39

it.   dist      |Z|       |J|        time in iteration
 0    0.31822  0.3161  0.3161   0:00:00.014883
 1    0.036317     0.0     0.0   0:00:00.017700
D(FCI, CCD manifold) = 0.03631677

Memory usage:
   Max. allocation: 2.0 kB
   Max. used: 4.3203125 kB
Ending at 26 May 2021 - 20:39
Total time: 0:00:00.050493
