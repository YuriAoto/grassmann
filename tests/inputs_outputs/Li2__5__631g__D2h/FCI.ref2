Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__631g__D2h

Command:
../../../Grassmann --method CCSD_mani_minD --molpro_output FCI.out

Starting at 26 May 2021 - 20:40

it.   dist      |Z|       |J|        time in iteration
 0    0.31822   0.318  0.3182   0:00:00.020876
 1    0.0028443  0.002859  0.00283   0:00:00.026019
 2    4.0411e-06  4.063e-06  4.021e-06   0:00:00.025912
 3    3.2743e-12  3.293e-12  3.257e-12   0:00:00.025577
D(FCI, CCSD manifold) = 0.00000000

Memory usage:
   Max. allocation: 2.0 kB
   Max. used: 4.3671875 kB
Ending at 26 May 2021 - 20:40
Total time: 0:00:00.123986
