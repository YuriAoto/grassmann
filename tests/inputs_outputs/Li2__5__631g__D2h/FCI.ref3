Grassmann
Exploring the geometry of the electronic wave functions space
Yuri Aoto - 2018, 2019, 2020

Current git commit: d08f010b2b9fea125a521ca84f2602dc10a6d81e

Directory:
/home/yuriaoto/Documents/Codes/grassmann/tests/inputs_outputs/Li2__5__631g__D2h

Command:
../../../Grassmann --method CCSD_mani_minD --molpro_output FCI.out --cc_diag_hess

Starting at 26 May 2021 - 20:44

it.   dist      |Z|       |J|        time in iteration
 0    0.31822  0.3182  0.3182   0:00:00.017020
 1    0.00065945  0.0006599  0.0006599   0:00:00.014534
 2    2.3949e-05  2.396e-05  2.398e-05   0:00:00.014186
 3    8.6961e-07  8.708e-07  8.708e-07   0:00:00.014266
 4    3.1592e-08  3.163e-08  3.165e-08   0:00:00.014126
 5    1.1479e-09  1.15e-09  1.15e-09   0:00:00.014310
D(FCI, CCSD manifold) = 0.00000000

Memory usage:
   Max. allocation: 2.0 kB
   Max. used: 4.3671875 kB
Ending at 26 May 2021 - 20:44
Total time: 0:00:00.106454
