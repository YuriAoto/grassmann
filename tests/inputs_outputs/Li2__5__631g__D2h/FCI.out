
 Primary working directories    : /scr/yuriaoto/job.032979/molpro.tGgU7G8KBz
 Secondary working directories  : /scr/yuriaoto/job.032979/molpro.tGgU7G8KBz
 Wavefunction directory         : /scr/yuriaoto/job.032979/
 Main file repository           : /scr/yuriaoto/job.032979/molpro.tGgU7G8KBz/

 id        : yuriaoto

 Nodes             nprocs
 t-rex.ufabc.int.br    1

 Using customized tuning parameters: mindgm=8; mindgv=16; mindgc=1; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (933), CPU time= 0.00 sec
 ***,
 memory,40,m
 gprint,orbital,basis,civector
 gthresh,thrprint=0,printci=0.0
 file,3,wfu.dat,old
 
 {fci
 wf,6,1,0;
 }
 Commands  initialized (760), CPU time= 0.01 sec, 617 directives.
 Default parameters read. Elapsed time= 0.04 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2019.2 linked Abr  7 2019 15:10:35


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit serial version                                                                  DATE: 26-May-21          TIME: 20:31:58  
 **********************************************************************************************************************************

 SHA1:             3a8a58c585c929645742fb5a1b7541a4ad1da699
 **********************************************************************************************************************************


 THRESHOLDS:

 ZERO    =  1.00D-12  ONEINT  =  1.00D-12  TWOINT  =  1.00D-11  PREFAC  =  1.00D-14  XXXXXX  =  1.00D-09  EORDER  =  1.00D-04
 ENERGY  =  0.00D+00  ETEST   =  0.00D+00  EDENS   =  0.00D+00  THRDEDEF=  1.00D-06  GRADIENT=  1.00D-02  STEP    =  1.00D-03
 ORBITAL =  1.00D-05  CIVEC   =  1.00D-05  COEFF   =  1.00D-04  PRINTCI =  0.00D+00  PUNCHCI =  9.90D+01  OPTGRAD =  3.00D-04
 OPTENERG=  1.00D-06  OPTSTEP =  3.00D-04  THRGRAD =  1.00D-10  COMPRESS=  1.00D-11  VARMIN  =  1.00D-07  VARMAX  =  1.00D-03
 THRDOUB =  0.00D+00  THRDIV  =  1.00D-05  THRRED  =  1.00D-07  THRPSP  =  1.00D+00  THRDC   =  1.00D-10  THRCS   =  1.00D-10
 THRNRM  =  1.00D-08  THREQ   =  0.00D+00  THRDE   =  1.00D+00  THRREF  =  1.00D-05  SPARFAC =  1.00D+00  THRDLP  =  1.00D-07
 THRDIA  =  1.00D-10  THRDLS  =  1.00D-07  THRGPS  =  0.00D+00  THRKEX  =  0.00D+00  THRDIS  =  2.00D-01  THRVAR  =  1.00D-10
 THRLOC  =  1.00D-06  THRGAP  =  1.00D-06  THRLOCT = -1.00D+00  THRGAPT = -1.00D+00  THRORB  =  1.00D-06  THRMLTP =  0.00D+00
 THRCPQCI=  1.00D-10  KEXTA   =  0.00D+00  THRCOARS=  0.00D+00  SYMTOL  =  1.00D-06  GRADTOL =  1.00D-06  THROVL  =  1.00D-08
 THRORTH =  1.00D-08  GRID    =  1.00D-06  GRIDMAX =  1.00D-03  DTMAX   =  0.00D+00  THROCCDE=  1.00D-01  THROCCDE=  1.00D-01
 THRPRINT=  0.00D+00


 Variable memory set to   40000000 words,  buffer space 



 Permanent file  3  wfu.dat  assigned.  Implementation=df    Size= 14.08 MB


 PROGRAM * RESTART 


 Reading variables from file 3

 _FNUC           =      0.00000000
 _BASIS          =    6-31G
 _NELEC          =      6.00000000
 _PROGRAM        =    RHF-SCF
 _DMX_SCF        =      0.00000000
 _DMY_SCF        =      0.00000000
 _DMZ_SCF        =      0.00000000
 _HOMO           =      2.10000000
 _EHOMO          =     -0.17851736
 _LUMO           =      2.50000000
 _ELUMO          =      0.01777251
 _ENERGY         =    -14.86544674
 _ENERGY_METHOD  =    RHF
 _ENERGY_BASIS   =    6-31G
 _ENUC           =      1.80000000
 _EPS1(1:5)      =     -2.44083670    -0.17851736     0.06347959     0.19419351     0.22544151
 _EPS2(1:2)      =      0.04562594     0.20818036
 _EPS3(1:2)      =      0.04562594     0.20818036
 _EPS5(1:5)      =     -2.44048575     0.01777251     0.13971385     0.26575149     0.54809025
 _EPS6(1:2)      =      0.09250729     0.27049360
 _EPS7(1:2)      =      0.09250729     0.27049360
 _IPROC_MPP      =      0.00000000
 _IPROC_MPPX     =      0.00000000
 _ORBITAL        =   2100.30000000
 _STATUS         =      1.00000000
 _DATE           =    21-Oct-19
 _LASTORB        =    RHF
 _OUTPUT         =    /scr/yuriaoto/job.022400/RHF.xml
 _PGROUP         =    D2h 
 _TIME           =    10:18:24
 R               =      5.00000000

 Geometry written to block  1 of record 700


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL   RESTART
 CPU TIMES  *         0.13      0.02
 REAL TIME  *         0.17 SEC
 DISK USED  *        26.84 MB
 **********************************************************************************************************************************


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry LI     S 6-31G                selected for orbital group  1
 Library entry LI     P 6-31G                selected for orbital group  1


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Linear
 User-specified symmetry elements: X,Y,Z
 Symmetry elements: X,Y,Z
 Rotational constants:      20.8009354     20.8009354      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  LI1     3.00    0.000000000    0.000000000   -2.500000000
   2  LI2     3.00    0.000000000    0.000000000    2.500000000

 Bond lengths in Bohr (Angstrom)

 1-2  5.000000000
     ( 2.645886055)


 BASIS DATA

   Nr Sym  Nuc  Type         Exponents   Contraction coefficients

    1.1 Ag    1  1s         642.418910     0.002143
              2  1s          96.798515     0.016209
                             22.091121     0.077316
                              6.201070     0.245786
                              1.935118     0.470189
                              0.636736     0.345471
    2.1 Ag    1  1s           2.324918    -0.035092
              2  1s           0.632430    -0.191233
                              0.079053     1.083988
    3.1 Ag    1  1s           0.035962     1.000000
              2  1s    
    4.1 Ag    1  2pz          2.324918     0.008942
              2 -2pz          0.632430     0.141009
                              0.079053     0.945364
    5.1 Ag    1  2pz          0.035962     1.000000
              2 -2pz   
    1.2 B3u   1  2px          2.324918     0.008942
              2  2px          0.632430     0.141009
                              0.079053     0.945364
    2.2 B3u   1  2px          0.035962     1.000000
              2  2px   
    1.3 B2u   1  2py          2.324918     0.008942
              2  2py          0.632430     0.141009
                              0.079053     0.945364
    2.3 B2u   1  2py          0.035962     1.000000
              2  2py   
    1.5 B1u   1  1s         642.418910     0.002143
              2 -1s          96.798515     0.016209
                             22.091121     0.077316
                              6.201070     0.245786
                              1.935118     0.470189
                              0.636736     0.345471
    2.5 B1u   1  1s           2.324918    -0.035092
              2 -1s           0.632430    -0.191233
                              0.079053     1.083988
    3.5 B1u   1  1s           0.035962     1.000000
              2 -1s    
    4.5 B1u   1  2pz          2.324918     0.008942
              2  2pz          0.632430     0.141009
                              0.079053     0.945364
    5.5 B1u   1  2pz          0.035962     1.000000
              2  2pz   
    1.6 B2g   1  2px          2.324918     0.008942
              2 -2px          0.632430     0.141009
                              0.079053     0.945364
    2.6 B2g   1  2px          0.035962     1.000000
              2 -2px   
    1.7 B3g   1  2py          2.324918     0.008942
              2 -2py          0.632430     0.141009
                              0.079053     0.945364
    2.7 B3g   1  2py          0.035962     1.000000
              2 -2py   

 NUCLEAR CHARGE:                    6
 NUMBER OF PRIMITIVE AOS:          44
 NUMBER OF SYMMETRY AOS:           44
 NUMBER OF CONTRACTIONS:           18   (    5Ag  +    2B3u +    2B2u +    0B1g +    5B1u +    2B2g +    2B3g +    0Au  )
 NUMBER OF CORE ORBITALS:           2   (    1Ag  +    0B3u +    0B2u +    0B1g +    1B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF VALENCE ORBITALS:        8   (    2Ag  +    1B3u +    1B2u +    0B1g +    2B1u +    1B2g +    1B3g +    0Au  )


 NUCLEAR REPULSION ENERGY    1.80000000

 Eigenvalues of metric

         1 0.103E+00 0.180E+00 0.946E+00 0.140E+01 0.370E+01
         2 0.252E+00 0.273E+01
         3 0.252E+00 0.273E+01
         5 0.422E-02 0.112E+00 0.166E+00 0.962E+00 0.242E+01
         6 0.108E+00 0.907E+00
         7 0.108E+00 0.907E+00


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:       2829.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:       2829      RECORD LENGTH: 524288

 Memory used in sort:       0.56 MW

 SORT1 READ        8184. AND WROTE        2829. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC
 SORT2 READ        2829. AND WROTE        2829. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.01 SEC

 FILE SIZES:   FILE 1:    30.2 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:     34.4 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       29.97       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       INT   RESTART
 CPU TIMES  *         0.27      0.14      0.02
 REAL TIME  *         0.40 SEC
 DISK USED  *        49.44 MB
 **********************************************************************************************************************************


 PROGRAM * FCI (Full CI)     Author: P.J. Knowles, 1984


 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)

 Frozen orbitals:              2 (  1  0  0  0  1  0  0  0)
 Active orbitals:             16 (  4  2  2  0  4  2  2  0)
 Active electrons:             2
 Spin quantum number:          0.0
 Orbital pairs:                    32      16      16       8      24      16      16       8
 Strings:                           4       2       2       0       4       2       2       0
                                    4       2       2       0       4       2       2       0
 Determinants:                             48              32              32              16
                                           48              32              32              16

 Load integrals                1.0 sec
 Transform integrals           1.0 sec
 Storage for integrals:           3280
 Remaining memory:            39995084

 Core energy:             -14.27079082
 Run Hamiltonian processor DAVIDSON
 Symmetry:                   1
 Trial vector:                  0.00
 Result vector:                 0.00
 Maximum iterations:           90
 Convergence threshold:         0.0000100
 RHS vector:                    0.00
 Hamiltonian shift:             0.0000000
 Output threshold:              0.0000000
 Options:                        0
 Number of roots:             1

 Hamiltonian diagonaliser entered at time      1.03

 Initial configuration generated:
       1      1.0000000    -14.8654467

 It  Tr    CPU  Convergence   Energy       Pop     dE(next)

  1   1    1.0  1.00000000   -14.86544674
                                           0.33 -0.03022028 -0.02679626
  2   1    1.0  0.29481559   -14.89058265
                                           0.25 -0.00229843 -0.00189852
  3   1    1.0  0.05941419   -14.89226137
                                           0.29 -0.00034862 -0.00029087
  4   1    1.0  0.03069556   -14.89252516
                                           0.23 -0.00006962 -0.00005276
  5   1    1.0  0.01295954   -14.89258038
                                           0.29 -0.00002943 -0.00002376
  6   1    1.0  0.01047511   -14.89260708
                                           0.27 -0.00001106 -0.00000850
  7   1    1.0  0.00531566   -14.89261591
                                           0.23 -0.00000211 -0.00000166
  8   1    1.0  0.00241859   -14.89261772
                                           0.29 -0.00000055 -0.00000044
  9   1    1.0  0.00112869   -14.89261820
                                           0.42 -0.00000012 -0.00000009
 10   1    1.0  0.00056903   -14.89261829
                                           0.27 -0.00000002 -0.00000002
 11   1    1.0  0.00024518   -14.89261831
                                           0.42 -0.00000001 -0.00000000
 12   1    1.0  0.00011768   -14.89261832
                                           0.23 -0.00000000 -0.00000000
 13   1    1.0  0.00004783   -14.89261832
                                           0.33 -0.00000000 -0.00000000
 14   1    1.0  0.00002143   -14.89261832
                                           0.19 -0.00000000 -0.00000000
 15   1    1.0  0.00000817   -14.89261832


RESULTS FOR STATE  1.1
=====================

 Correlation energy                    -0.027171580482
 !FCI STATE  1.1 Energy               -14.892618319669
 CI Vector (threshold=  0.00    )
       0.952916487383  1  2  3  1  2  3
      -0.016018793126  1  2  3  1  2  4
       0.008418194279  1  2  3  1  2  5
       0.016472682994  1  2  3  1  2  6
      -0.016018793126  1  2  4  1  2  3
      -0.067262363304  1  2  4  1  2  4
       0.054180124375  1  2  4  1  2  5
       0.001353671387  1  2  4  1  2  6
       0.008418194279  1  2  5  1  2  3
       0.054180124375  1  2  5  1  2  4
      -0.052000638675  1  2  5  1  2  5
       0.002994092846  1  2  5  1  2  6
       0.016472682994  1  2  6  1  2  3
       0.001353671387  1  2  6  1  2  4
       0.002994092846  1  2  6  1  2  5
      -0.003836972862  1  2  6  1  2  6
      -0.136409096490  1  2  7  1  2  7
      -0.066251891026  1  2  7  1  2  8
      -0.066251891026  1  2  8  1  2  7
      -0.035157406178  1  2  8  1  2  8
      -0.136409096490  1  2  9  1  2  9
      -0.066251891026  1  2  9  1  2 10
      -0.066251891026  1  2 10  1  2  9
      -0.035157406178  1  2 10  1  2 10
      -0.099943800807  1  2 11  1  2 11
       0.059093825538  1  2 11  1  2 12
      -0.003824530391  1  2 11  1  2 13
       0.024359476783  1  2 11  1  2 14
       0.059093825538  1  2 12  1  2 11
      -0.037681512734  1  2 12  1  2 12
       0.002224463901  1  2 12  1  2 13
      -0.017419201664  1  2 12  1  2 14
      -0.003824530391  1  2 13  1  2 11
       0.002224463901  1  2 13  1  2 12
       0.000031902311  1  2 13  1  2 13
       0.001042992927  1  2 13  1  2 14
       0.024359476783  1  2 14  1  2 11
      -0.017419201664  1  2 14  1  2 12
       0.001042992927  1  2 14  1  2 13
      -0.011068858633  1  2 14  1  2 14
       0.000817200650  1  2 15  1  2 15
      -0.000917493183  1  2 15  1  2 16
      -0.000917493183  1  2 16  1  2 15
      -0.002977990215  1  2 16  1  2 16
       0.000817200650  1  2 17  1  2 17
      -0.000917493183  1  2 17  1  2 18
      -0.000917493183  1  2 18  1  2 17
      -0.002977990215  1  2 18  1  2 18
 /EOF
 !FCI STATE  1.1 Dipole moment          0.00000000     0.00000000     0.00000000
 Dipole moment /Debye                   0.00000000     0.00000000     0.00000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       29.97       500      700      610      900      950      970     1000      129      960     1100   
                                          VAR     GEOM    BASINP   SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              3       5       14.77       500      610      700     1000     2100   
                                          VAR    BASINP    GEOM     BASIS     RHF  

 PROGRAMS   *        TOTAL       FCI       INT   RESTART
 CPU TIMES  *         0.28      0.01      0.14      0.02
 REAL TIME  *         0.41 SEC
 DISK USED  *        49.44 MB
 **********************************************************************************************************************************
 Molecular orbitals read from record     2100.3  Type=RHF/CANONICAL (state 1.1)
 Occupation numbers read from record     2100.3  Type=RHF/RHF (state 1.1)
 Orbital energies read from record       2100.3  Type=RHF/CANONICAL (state 1.1)
 Redundancy group numbers read from rec  2100.3  Type=RHF/RHF (state 1.1)

 FCI/6-31G energy=    -14.892618319669

             FCI
    -14.89261832
 **********************************************************************************************************************************
 Molpro calculation terminated
